src/garage/envs/__init__.py

```diff
@@ -6,6 +6,7 @@
 from garage.envs.grid_world_env import GridWorldEnv
 from garage.envs.normalized_env import normalize
 from garage.envs.point_env import PointEnv
+from garage.envs.half_cheetah_vel import HalfCheetahVelEnv
```

Comment:
can you break this env into a separate PR?

Reply:
I'm working on a PR to add this (and `HalfCheetahDirEnv`).